icevision/models/mmdet/common/bbox/single_stage/model.py

```diff
@@ -5,26 +5,49 @@
 from mmdet.models import build_detector
 from mmcv.runner import load_checkpoint
 from icevision.models.mmdet.utils import *
+from icevision.utils.download_utils import *
 
 
 def model(
-    cfg: Union[str, Path, Config],
+    backbone,
     num_classes: int,
-    weights_path: Optional[Union[str, Path]] = None,
+    pretrained: bool = True,
+    checkpoints_path: Optional[Union[str, Path]] = None,
```

Comment:
instead of having `None` here you can have `'checkpoints` and you avoid the `if checkpoints_path` down there